### ISSUE TITLE
Mazurka effect should wear on performing an offensive action

### ISSUE THREAD

**Issue by isotor** [DarkstarProject/darkstar#5855]
**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:** 30190328_2

**_Source Branch_** (master/stable) **:** master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

Excerpt from the relevant [wiki page](https://ffxiclopedia.fandom.com/wiki/Mazurka):

> - Mazurka effect immediately wears off if the player performs any sort of offensive action on an enemy, or if an enemy performs any sort of offensive action on the player. This is designed to inhibit the effectiveness of kiting tactics.

**Comment by KnowOne134** [comment]
Does this also effect all mivement spells and abilities? Cor and dnc have thier own version

**Comment by isotor** [comment]
Good point - yes this should also be the case also for [Chocobo Jig](https://ffxiclopedia.fandom.com/wiki/Chocobo_Jig) and [Bolter's Roll](https://ffxiclopedia.fandom.com/wiki/Bolter%27s_Roll). Unfortunately I built the server with the setting toggled off for anything post-Abby so can't check those right now - if nobody else can give this a quick look I'll rebuild and test in the next few days.

**Comment by TeoTwawki** [comment]
its the diff between Flee-type effects and Quickening-type effects.
one stays, one doesn't.
